src/core/client/admin/routes/Moderate/Queue/ForReviewQueueRoute/ForReviewQueueRow.tsx

```diff
@@ -0,0 +1,159 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+import CommentParser from "coral-admin/components/Comment/CommentParser";
+import { getLocationOrigin } from "coral-framework/utils";
+import { CheckBox, Flex, TableCell, TableRow } from "coral-ui/components/v2";
+import { TimestampFormatter } from "coral-ui/components/v2/Timestamp";
+import { ComponentLink } from "coral-ui/components/v3";
+
+import { COMMENT_FLAG_REASON } from "coral-admin/__generated__/ForReviewQueueRoute_query.graphql";
+
+import styles from "./ForReviewQueueRow.css";
+
+interface ForReviewRevision {
+  readonly body: string | null;
+}
+
+interface ForReviewComment {
+  id: string;
+  revision: ForReviewRevision;
+}
+
+interface Props {
+  id: string;
+  comment: ForReviewComment | null;
```

Comment:
Comment is never `null` so we don't need to handle this case.

Reply:
Could also ask for what is actually needed instead of nesting an object.
```
commentID: string
commentBody: string
``

Reply:
Would all change anyway if turned into a Container, see: https://github.com/coralproject/talk/pull/3566#pullrequestreview-668244008

Reply:
Correct, fixed, patched, all refactored away with container